app/test_main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+import pytest
+
+from unittest import mock
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "mock_url, mock_internet, expected_result",
+    [
+        pytest.param(True, True, "Accessible", id="Access provided"),
+        pytest.param(
+            True,
+            False,
+            "Not accessible",
+            id="Access denied due to lack of internet",
+        ),
+        pytest.param(
+            False,
+            True,
+            "Not accessible",
+            id="Access denied due to unsatisfactory status code",
+        ),
+    ],
+)
```

Comment:
I think, one more test should be added when mock_url = False,  mock_internet = False

Reply:
I thought about it, but i'm not sure it's necessary, cause we already have two tests, which test both conditions separately, and one more test, which confirms that both conditions should be executed, but thank you for your advice)🙃

Reply:
Agree, it's better to add such test to be sure that the function works correctly